NEXT-39316 - MySQL cache invalidator adapter (#1628)
shopware/docs: guides/hosting/performance/performance-tweaks.md

### Delayed invalidation

A delay for cache invalidation can be activated for systems with a high update frequency for the inventory (products, categories). Once the instruction to delete the cache entries for a specific product or category occurs, they are not deleted instantly but processed later by a background task. Thus, if two processes invalidate the cache in quick succession, the timer for the invalidation of this cache entry will only reset.
By default, the scheduled task will run every 20 seconds, but the interval can be adjusted over the `scheduled_taks` DB table, by setting the `run_interval` to the desired value (it is configured in seconds) for the entry with the name `shopware.invalidate_cache`.

::: warning
If you enable delayed cache invalidation, you must set up a worker to run [Scheduled Tasks](../infrastructure/scheduled-task), e.g., using the [Message Queue](../infrastructure/message-queue).
:::

> There are two possible storages/adapters to use for delayed cache invalidation: Redis & MySQL. Redis is preferred since it handles retrieving and deleting keys in an atomic manner. MySQL also supports it, but its more complicated and at a certain load its inevitable that deadlocks will occur. If you already use Redis, use it also for the delayed cached. The MySQL adapter should only be used when you cannot use Redis.

Reviewer: ```suggestion
There are two possible storages/adapters for delayed cache invalidation: Redis and MySQL. Redis is preferred since it handles retrieving and deleting keys in an atomic manner. MySQL also supports it, but it's more complicated, and at a certain load, deadlocks are inevitable. If you already use Redis, use it also for the delayed cached. The MySQL adapter should only be used when you cannot use Redis.
```
